src/main/fc/fc_init.c

```diff
@@ -156,6 +160,18 @@ void processLoopback(void)
 }
 
 
+#ifdef VTX_RTC6705
+bool canUpdateVTX(void)
```

Comment:
alas no, it's needed by other parts of the system and is annoying because it's not OSD or VTX specific.  It's only needed by boards that have a device on the same bus as the vtx or some other constraint on the ability to communicate with the VTX.  A better location for the code would be good but I've struggled a few times on deciding where this small bit of code should live and what the dependencies should look like.